cocos/renderer/gfx-metal/MTLTexture.mm

```diff
@@ -80,12 +80,12 @@
             descriptor = [MTLTextureDescriptor texture2DDescriptorWithPixelFormat:mtlFormat
                                                                             width:_width
                                                                            height:_height
-                                                                        mipmapped:_flags & GFXTextureFlags::GEN_MIPMAP];
+                                                                        mipmapped:NO];
```

Comment:
Why change to NO?

Reply:
It's duplicated since mipmap level count was set here:
https://github.com/cocos-creator/cocos2d-x-lite/blob/70eff6363b685056831f59d8691b7b33a75499f9/cocos/renderer/gfx-metal/MTLTexture.mm#L100

Reply:
Good to know. Could you please add some comment for it?